src/tests/test_trivia.py

```diff
@@ -0,0 +1,36 @@
+import os
+import json
+import pytest
+from ..service.trivia import TriviaService
+
+@pytest.fixture(name="trivia",autouse=True)
+def fixture_trivia():
+    # setUp TriviaService
+    return TriviaService()
+
+@pytest.fixture(name="file_trivia",autouse=True)
+def fixture_file_trivia():
+    # SetUp trivias from file
+    filename = os.path.join(os.path.dirname(__file__),'../json/trivia.json')
+    with open(file=filename, encoding="utf-8") as f:
+        data = f.read()
+        return json.loads(data)['trivia']
```

Comment:
This can be simplified
```suggestion
        return json.load(f)['trivia']
```